### ISSUE TITLE
After schedule.disable all jobs reported individually disabled intead of just the scheduler

### ISSUE THREAD

**Issue by OrangeDog** [saltstack/salt#54848]
### Description of Issue
```
# salt-call schedule.list
local:
    schedule:
      enabled: true
      highstate:
        enabled: true
        function: state.apply
        jid_include: true
        kwargs:
          saltenv: base
        maxrunning: 1
        minutes: 15
        name: highstate
        returner: zabbix
        run_on_start: true
        splay: 60
# salt-call schedule.disable
local:
    ----------
    comment:
        Disabled schedule on minion.
    result:
        True
# salt-call schedule.list
local:
    schedule:
      enabled: false
      highstate:
        enabled: false
        function: state.apply
        jid_include: true
        kwargs:
          saltenv: base
        maxrunning: 1
        minutes: 15
        name: highstate
        returner: zabbix
        run_on_start: true
        splay: 60
```

Subsequent operations don't put them back
```
# salt-call schedule.enable
local:
    ----------
    comment:
        Enabled schedule on minion.
    result:
        True
# salt-call schedule.list
local:
    schedule:
      enabled: true
      highstate:
        enabled: false
        function: state.apply
        jid_include: true
        kwargs:
          saltenv: base
        maxrunning: 1
        minutes: 15
        name: highstate
        returner: zabbix
        run_on_start: true
        splay: 60
```

Related to #54820 
https://gist.github.com/OrangeDog/fd08470a682752347efd52fe7a5b830d

**Comment by OrangeDog** [comment]
Additionally, if this is eventually persisted via a `schedule.present` application, the `__mine_interval` job remains disabled and I'm not sure there's any way to turn it back on other than editing the file and restarting the minion.

Related: #54847

**Comment by OrangeDog** [comment]
Fixed in 2019.2.2
